javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+var message2 = document.querySelector('.countdown-display');
+
+var text = ['3', '2', '1', '~Earth Beeeeloooww Us!'];
+
+var count = 0;
+
+function update() {
+  message2.innerHTML = text[count];
```

Comment:
It's usually a bad idea to set the `innerHTML` property of a DOM element directly, for both speed and security reasons. How else could you change the text within the `.countdown-display` element?